src/ansiblelint/app.py

```diff
@@ -138,11 +147,19 @@ def report_outcome(
         #         v,
         #     )
 
-        if result.matches and not self.options.quiet:
+        if self.options.write and "yaml" in self.options.skip_list:
+            console_stderr.print(
```

Comment:
Please use logging.warning for this as it gives more control.

In fact I doubt we ever need to use console_stderr other than with logging.

Reply:
done